highlevel/robot/gateway/motor_test.py

```diff
@@ -14,12 +17,12 @@ class TestMotorGateway:
     """
     @staticmethod
     @pytest.mark.asyncio
-    async def test_set_speed(socket_adapter_mock):
+    async def test_set_speed(socket_adapter_mock, configuration_test):
         """
-        Test that the motor gateway encodes a wheel speed message.
+        Test that the motor gateway encodes a wheel target speed message.
         """
-        motor_gateway = MotorGateway(motor_board_adapter=socket_adapter_mock)
-        await motor_gateway.set_speed(10, 20)
+        motor_gateway = MotorGateway(socket_adapter_mock, configuration_test)
```

Comment:
use KW values when calling constructors,

Reply:
We would have to go through all the repo, as it is not done everywhere.
We can refactor that in a future clean.

Reply:
We usually use KW arguments for calling constructors.

Anyway, here you are explicitely **removing** the kw arguments, please do not do that, that's a regression